### ISSUE TITLE
Dashboard loading appearance

### ISSUE THREAD

**Issue by 0nko** [woocommerce/woocommerce-android#1211]
While I was working on the enhanced empty view, @anitaa1990 raised a good point about the loading state of the dashboard. It currently shows the empty dashboard layout which is eventually populated with data. But this looks a bit confusing if a store doesn't have any orders:

<img width="300" src="https://user-images.githubusercontent.com/22608780/60573461-55ff6680-9d95-11e9-8645-1aaf64f62222.gif">

It looks like there is something and then it's suddenly hidden. I'm not sure what an ideal solution would be here. This is just to start a discussion, feel free to close this.

**Comment by AmandaRiu** [comment]
Yeah this has bugged me for a while now. What if we were to have a basic pre-loading view that just shows the gray background with an indeterminate progress (circular) in the center. Once we have a single return on a stat that has data, we switch the dashboard loading skeleton. If we get back no stats, then we can immediately show the "Share your store" view. @kyleaparker - thoughts?

**Comment by kyleaparker** [comment]
I shared a screenshot on this issue over here https://github.com/woocommerce/woocommerce-android/pull/1202#issuecomment-508255546
I think they should still land on the dashboard we would just alter which cards are shown until they receive their first order, so ideally they wouldn't see the chart skeleton on the dashboard or top performers

**Comment by AmandaRiu** [comment]
@0nko  since this is already a work in progress, I'm assigning it to the current milestone. If it doesn't get done we can move it to the next milestone.

**Comment by JorgeMucientes** [comment]
Closing this as part of a maintenance pass on the GitHub issue backlog. This issue matched the auto-close criteria for this pass: it has not been updated since 2021-05-06, is unassigned, is not labeled as a bug or high/medium priority, and does not appear to involve POS, payments, login, data loss, crash, security, vulnerability, auth, or fraud.

If this is still relevant, please comment with current details and we can reopen it or move it into Linear for triage.
